Core/GDCore/Events/InstructionsList.h

```diff
@@ -35,17 +36,17 @@ class InstructionsList : public SPtrList<gd::Instruction> {
    * \brief Serialize the instructions to the specified element
    * \see EventsListSerialization
    */
-  void SerializeTo(gd::SerializerElement& element) const;
+  void SerializeTo(gd::SerializerElement &element) const;
 
   /**
    * \brief Load the instructions from the specified element
    * \see EventsListSerialization
    */
-  void UnserializeFrom(gd::Project& project,
-                       const gd::SerializerElement& element);
+  void UnserializeFrom(gd::Project &project,
+                       const gd::SerializerElement &element);
```

Comment:
Why these signature changes?

Reply:
Autoformatter

Reply:
Note: we have two diverging rules according to files, it will be good at some point to force a single style and clang-format all C++ files in a PR.